CHANGELOG.rst

```diff
@@ -20,6 +20,8 @@ This document describes changes between each past release.
 - Fix group association when members contains ``system.Authenticated`` (fixes #776)
 - Raise an error when members contains ``system.Everyone`` or a group ID (#850)
 - Fix StatsD view counter with 404 responses (#853)
+- Fixes filtering when the parameter is zero. (fixes #851)
```

Comment:
Better «*when filtering on id with numeric values*» or something like that, no ?